### PR TITLE
Write tests for createFullPaths

### PR DESCRIPTION
**Pull request by RossWebFS**

### CODE REVIEW

File: tests/unit/src/utils/createFullRoutePaths.spec.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  bookingForm: {
+    path: "booking-form",
+    details: {
+      path: ":treatmentId",
+    },
+  },
+  any: {
+    path: "*",
+  },
+};
```
[the remark this answers is not in the document]
Comment (RossWebFS) [reply tests/unit/src/utils/createFullRoutePaths.spec.ts:19]: You are about right. I'll try on this case

File: tests/unit/src/utils/createFullRoutePaths.spec.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+const routePaths = createFullRoutePaths(pathsTree) as typeof pathsTree;
+
+describe("createFullRoutePaths", () => {
```
[the remark this answers is not in the document]
Comment (RossWebFS) [reply tests/unit/src/utils/createFullRoutePaths.spec.ts:23]: Thanks for correcting me. I'll follow this convention next time

File: tests/unit/src/utils/createFullRoutePaths.spec.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    );
+  });
+  it("should return correct path for non-existed route", () => {
+    const routePaths = createFullRoutePaths(pathsTree) as Record<
+      string,
+      { path: string }
+    >;
+
+    expect(routePaths.nlkn).toBe("*");
+    expect(routePaths.any?.path).toBe("*");
```
[the remark this answers is not in the document]
Comment (RossWebFS) [reply tests/unit/src/utils/createFullRoutePaths.spec.ts:43]: Yes. That the best solution I come up with

File: tests/unit/src/utils/createFullRoutePaths.spec.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (RossWebFS) [reply tests/unit/src/utils/createFullRoutePaths.spec.ts:34]: Fixed

**Review by mhevyk — APPROVED** [review]

**Pull request merged**